src/helpers/lib/formatBN.js

```diff
@@ -0,0 +1,17 @@
+const BN = require('bn.js')
+
+exports.tenPow = x => (
+  (new BN(10)).pow(new BN(x))
+)
+
+exports.formatBN = (value, base, precision) => {
+  // Inspired by: https://github.com/ethjs/ethjs-unit/blob/35d870eae1c32c652da88837a71e252a63a83ebb/src/index.js#L83
+  const baseLength = base.toString().length
+
+  let fraction = value.mod(base).toString()
+  const zeros = '0'.repeat(Math.max(0, baseLength - fraction.length - 1))
+  fraction = `${zeros}${fraction}`
+  const whole = value.div(base).toString()
+
+  return `${whole}${parseInt(fraction) === 0 ? '' : `.${fraction.slice(0, precision)}`}`
```

Comment:
It shouldn’t be a problem in modern JS engines, but it can still be a good idea to specify the radix as a second parameter of `parseInt()` for a while, just in case scripts are run on unsupported engines (e.g. old mobile browser or Node.js version) https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt#Octal_interpretations_with_no_radix